src/ray/util/logging.cc

```diff
@@ -137,8 +138,13 @@ void RayLog::StartRayLog(const std::string &app_name, RayLogLevel severity_thres
         app_name_without_path = app_name.substr(pos + 1);
       }
     }
+    google::InitGoogleLogging(app_name_.c_str());
```

Comment:
Why do we not need `google::InitGoogleLogging` when `log_dir.empty()` now?
Also, it seems that `log_dir` is always empty. 
Am I missing something?

Reply:
`google::InitGoogleLogging` resulting in storing logs in `/tmp`, in the meanwhile, all the logs will be output to stderr and the python code (services.py) will redirect the logs to `/tmp/ray`. 
So if we call `google::InitGoogleLogging` when `log_dir.empty()`,  same logs would be stored both in `/tmp` and `/tmp/ray` which increasing storage overhead. 
`log_dir` is always empty, so duplicated logs should be stored unless we call `StartRaylog` with parameter of non-empty `log_dir`.